src/main/java/com/example/travelservice/integration/WeatherFromApi.java

```diff
@@ -0,0 +1,34 @@
+package com.example.travelservice.integration;
+
+import com.example.travelservice.dto.WeatherDto;
+import com.example.travelservice.model.Location;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class WeatherFromApi {
```

Comment:
please rename it, all services should end on "Service", for example WeatherIntegrationService

Reply:
done